pay-queue/wsgi.py

```diff
@@ -11,9 +14,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Enum definitions."""
+"""Provides the WSGI entry point for running the application."""
+import os
+from pay_queue import create_app
+
+
+app = create_app()
```

Comment:
We called this `app.py` in auth, hmm maybe we should make it consistent?

Reply:
fixed